app/policies/realtime_data/sip_options_prober_policy.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+module RealtimeData
+  class SipOptionsProberPolicy < ::RolePolicy
+    section 'RealtimeData/SipOptionsProberPolicy'
```

Comment:
when you add new section to policy you need to ad it to `config/policy_roles.yml.distr` and to your local `nfig/policy_roles.yml`